src/kicker/main.c

```diff
@@ -46,27 +63,43 @@ volatile uint8_t kick_on_breakbeam_strength_ = 0;
 
 unsigned ball_sense_change_count_ = 0;
 
-unsigned time = 0;
+uint32_t time = 0;
```

Comment:
Wait Seriously. How is this updated? I can't find it anywhere.

Reply:
That is a legitimate bug actually, time++ disappeared somewhere, so I guess we're constantly calling the ADC read which could throw off some timing in the main loop. Thanks!